protocol/buff/reader.go

```diff
@@ -25,19 +25,19 @@ import (
 
 // Message is a Reader with a Type attribute.
 type Message struct {
-	bts  []byte
+	Bts  []byte
```

Comment:
I'd call it `Payload`, but I hope we'll be able to remove this struct in future PRs.

Reply:
I was thinking of merging Message into the writer so that there is just a buff.Buff